x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/result_settings_non_text_fields_header.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiTableHeader, EuiTableHeaderCell } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export const ResultSettingsNonTextFieldsHeader: React.FC = () => {
+  return (
+    <EuiTableHeader className="resultSettingsTable__nonTextHeader">
+      <EuiTableHeaderCell align="left">
+        {i18n.translate(
+          'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.column.nonTextFieldsTitle',
+          { defaultMessage: 'Non-text fields' }
+        )}
+      </EuiTableHeaderCell>
+      <EuiTableHeaderCell align="center">
+        {i18n.translate(
+          'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.column.rawTitle',
+          { defaultMessage: 'Raw' }
+        )}
+      </EuiTableHeaderCell>
+      <EuiTableHeaderCell colSpan={4} />
```

Comment:
Not a change request / we can revisit this later when doing a general accessibility pass, but as a heads up it looks like screen readers are still picking up on the empty columns so we'd likely need to throw an `aria-hidden={true}` on here

Reply:
Sorry, you know, I actually didn't go through and test this with as screen reader. I sort of just assumed things would work, which was a bad assumption. Will test it next time: e76a5445a8d4bef69bb6105acae79d045ac8b03c

Reply:
All good!! I actually assumed the same myself especially about empty rows so this was news to me. FWIW I only tested out of curiosity because I was wondering how screen readers would handle the multiple `thead`s lol